createurl.php

```diff
@@ -31,6 +31,8 @@ function bve($bv) {
     }
 }
 
+$_POST['urlr'] = $_POST['naked'] . "/?utm_source=" . $_POST['utm-source'] . "&utm_medium=" . $_POST['utm-medium'] . "&utm_campaign=" . $_POST['utm-name'];
```

Comment:
Good point. Also,  maybe we could phase something in like this as a plugin rather than a core feature.  We generally want a light and minimalist system rather than a more complicated one.